app-games/gamemode/autobuild/templates

```diff
@@ -0,0 +1,28 @@
+Template: gamemode/add_user_to_group
+Type: note
+Description: Gamemode User Group Configuration
+ To enable all features of 'gamemode', add the user to the 'gamemode' group.
```

Comment:
```
To enable all GameMode features, add your user to the `gamemode' group with the following command:
.
sudo usermod -aG gamemode <username>
.
After adding your user to the group, please re-login for the changes to take effect.
.
You may also enable GameMode automatically as you log in with the following command:
.
systemctl --user enable gamemoded
```

No need for `--now` as it will take effect with a relogin.